external/implementations/samlconf-openam-impl/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2018 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+// Build file
+group = "org.codice.samlconf.implementations"
+description = "implementation for OpenAM IDP"
+
+dependencies {
+
+    compile(Libs.gson)
+    compile(Libs.kaptMetainfService)
+    compile(Libs.cxfSsoSaml)
+
+    compile(project(":ctk:common"))
```

Comment:
❓ Do we not need `compile(project(":external:samlconf-plugins-api"))` where the interface is?

Reply:
Should need it. I think you might fail on a clean repository build without it.